tests/MongoDB.Driver.Core.TestHelpers/JsonDrivenTests/CommandStartedEventAsserter.cs

```diff
@@ -110,6 +110,7 @@ private void AssertCommandAspect(BsonDocument actualCommand, string name, BsonVa
             {
                 switch (name)
                 {
+                    case "allowDiskUse":
```

Comment:
Our test runners were already treating `null` to mean "this field must be omitted"...

... but subject to a whitelist of which field names are meant to be asserted against in that way.

Reply:
Ack